### ISSUE TITLE
Add a section about how training data resizing might affect the quality of the end models

### ISSUE THREAD

**Issue by sayakpaul** [huggingface/diffusers#6397]
Have been chatting with @bghira on our Discord forum about the negative effects of not carefully resizing the training data. 

@bghira noticed that the way we resize the images in our training examples can introduce unwanted artifacts in the samples generated by the end fine-tuned model. In short, it ruins the fidelity of the generated samples. 

We don't want to make the data input pipelines in our training examples super sophisticated, guaranteeing SoTA results. But that said, I think it's best in the community's interest if we made a note about this phenomenon in the READMEs and just added a comment about it in the pipelines. 

Curious to know what @patil-suraj @apolinario think about this. Personally, I think it makes sense. 

I will let @bghira share some more insights as well.

**Comment by sayakpaul** [comment]
Not stale.

**Comment by asomoza** [comment]
is this still an issue @sayakpaul? I think the community is already aware of this and as always, the best solution is to have a ready made dataset with the optimal resolutions for training instead of doing the resize on the fly while training.

Also newer models don't seem to be affected by this that much or at least that's what I have experienced and saw, some people are even training Flux with 512px images.

**Comment by bghira** [comment]
resizing artifacts are learnt by Flux as well, and nearest neighbour resize done automatically is still the way the training examples here run so they're not setting a good example for others' best practices, this is still something to address

**Comment by asomoza** [comment]
you are do a lot of training so I'll believe you, I'll try to address this in the training examples then by adding a note.

**Comment by bghira** [comment]
i think a while back the image interpolation was going to be made configurable with the LANCZOS as the default instead, if you prefer

**Comment by crapthings** [comment]
What do the 'unwanted artifacts' look like?

**Comment by bghira** [comment]
https://zuru.tech/blog/the-dangers-behind-image-resizing

**Comment by bghira** [comment]
@stevhliu maybe since you're the docs expert we can bypass process and start working on what this would look like? here's the reference from Zuru tech blog;

![Image](https://github.com/user-attachments/assets/c65e8423-e9c4-4a71-ab1f-6147464d161a)

**Comment by stevhliu** [comment]
Yeah that sounds good to me. Maybe we can make a note of it in the training [Overview](https://huggingface.co/docs/diffusers/main/en/training/overview) since that is also where we introduce our training scripts.

Would you like to open a PR for this @bghira or I can too once I've wrapped up some other things 🙂
